src/test/java/org/openmrs/contrib/qaframework/automation/LocationManagementSteps.java

```diff
@@ -0,0 +1,153 @@
+import org.openmrs.contrib.qaframework.page.AddEditLocationPage;
+import org.openmrs.contrib.qaframework.page.ManageLocationTagsPage;
+import org.openmrs.contrib.qaframework.page.ConfigureMetadataPage;
+
+
+public class LocationManagementSteps extends Steps {
+
+    private static final String HANDLER_CONFIGURATION = "I have entered the handler configuration";
+    private static final String DATATYPE_CONFIGURATION = "I have entered the data configuration";
+    private static final String TAG_NAME = "This is a tag name";
+    private static final String MAX_VALUE = "20";
+    private static final String MIN_VALUE = "1";
+    private static final String DESCRIPTION = "This is the description";
+    private static final String NAME = "Noah";
+    private static final String LOCATION_NAME = "Bugema";
+    private static final String ADDRESS_ONE = "Bugema one";
+    private static final String ADDRESS_TWO = "Bugema two";
+    private static final String CITY = "Kampala";
+    private static final String STATE = "Central";
+    private static final String COUNTRY = "Uganda";
+    private ConfigureMetadataPage configureMetadataPage;
+    private LocationPage locationPage;
+    private ManageLocationsPage manageLocationsPage;
+    private AddEditLocationPage addEditLocationPage;
+    private ManageLocationTagsPage manageLocationTagsPage;
+    private ManageLocationAttributeTypesPage manageLocationAttributeTypesPage;
+
+    @Before(RunTest.HOOK.SELENIUM_LOCATION_MANAGEMENT)
+    public void visitHomePage() {
+        initiateWithLogin();
+    }
+
+    @After(RunTest.HOOK.SELENIUM_LOCATION_MANAGEMENT)
+    public void destroy() {
+        quit();
+    }
+
+    @When("a user clicks on the configure metadata link from the home page")
+    public void clicksConfigureMetaDataLink(){
+        configureMetadataPage = homePage.goToConfigureMetadata();
+    }
+
+    @Then("the system loads configure metadata link dashboard")
+    public void loadsConfigureMetaData(){
+        assertTrue(textExists("Configure Metadata"));
+    }
+
+    @And("a user clicks on Manage Location Attribute Types")
+    public void clickManageLocationAttributesTypes(){
+        manageLocationAttributeTypesPage = configureMetadataPage.goToManageLocationAttributeTypesPage();
+    }
+
+    @And("a user clicks on add new location attribute type")
+    public void addNewLocationAttributeType(){
+        manageLocationAttributeTypesPage.goToAddNewLocationAttributeTypeForm();
+    }
+
+    @And("a user fills the form")
+    public void fillsTheForm(){
+        manageLocationAttributeTypesPage.enterLocationAttributeTypeName(NAME);
+        manageLocationAttributeTypesPage.enterLocationAttributeTypeDescription(DESCRIPTION);
+        manageLocationAttributeTypesPage.enterMinimumOccurs(MIN_VALUE);
+        manageLocationAttributeTypesPage.enterMaximumOccurs(MAX_VALUE);
+        manageLocationAttributeTypesPage.selectDatatype("Location Datatype");
```

Comment:
the element here (line 88) is causing this build failure as shown in this [cucumber report](https://reports.cucumber.io/reports/98c224ae-483b-4c9e-a35f-9c6abc85fc57). take a look and ensure the cssSelector is up-to-date with the newly released refapp version

Reply:
> 

@mherman22 I tried to change it to an id to see what will though the cssSelector is upto date